src/Sulu/Bundle/AdminBundle/Resources/js/components/Dialog/Dialog.js

```diff
@@ -56,6 +64,10 @@ class Dialog extends React.Component<Props> {
                 this.open = open;
             }));
         }
+
+        if (snackbarMessage && this.snackbarType !== snackbarType) {
+            this.snackbarType = snackbarType;
```

Comment:
whats the reason for storing this in a class variable instead of using the `snackbarType` prop directly? 🤔

Reply:
Let's assume the dialog renders a warning. When the user clicks on the snackbar, the `onSnackbarClick()` prop will be called. Then, the container will probably do something like resetting it's `warning` state. From that moment, the container will pass `undefined` for both, `snackbarType` and `snackbarMessage` props to the dialog. But as soon as `snackbarType` is `undefined`, the snackbar would turn red (because the default is `error`) while still swiping up

Reply:
makes sense - i think there is a similar scenario for the `message` of the snackbar that is already handled in the `Snackbar` component:
https://github.com/sulu/sulu/blob/0238c55bd6a4d965aa7ebd12b2ba7467f33db957/src/Sulu/Bundle/AdminBundle/Resources/js/components/Snackbar/Snackbar.js#L42-L44

what do you think about moving this check into the `Snackbar` component too? that way both checks would be in the same place nad it might be easier to understand why this is needed 🙂